app/controllers/locations_controller.rb

```diff
@@ -3,7 +3,11 @@ class LocationsController < ApplicationController
 
   # GET /locations
   def index
-    @locations = Location.all
+    if user_signed_in? && current_user.is_host
+      @locations = Location.find_by(user: current_user)
```

Comment:
Just encountered a small bug with Laura - the `.find_by` method returns the first object it finds, so we should use `.where` instead, which returns a collection (an array)
```suggestion
      @locations = Location.where(user: current_user)
```

Reply:
Suggested change was implemented. Thank you!